src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,67 @@
+package mate.academy.model;
+
+import com.sun.istack.NotNull;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    @Column(length = 6555)
+    @NotNull
+    private String email;
+    @Column(length = 67676)
+    @NotNull
+    private String password;
+    private byte [] salt;
+
+    public User() {
+    }
```

Comment:
We don't need do declare empty constructor when it is the only one constructor in class. It will be generated by default